### ISSUE TITLE
Add ability to disable relationships

### ISSUE THREAD

**Issue by dusktreader** [marshmallow-code/marshmallow-sqlalchemy#98]
The include_fk option is really nice when producing flat serializations (non-nested). However, there's the issue of the foreign keys showing up twice: once as the proper foreign key name, and once as the relationship from the sqlalchemy model.

Please add either another option to exclude relationships or to have the include-fk option automatically get rid of simple relationship attributes that are just wrapping a foreign key

**Comment by diogommartins** [comment]
`marshmallow-sqlachemy` converts sqlalchemy's relationships to `Related` fields (you can confirm that behaviour at `marshmallow_sqlalchemy.convert.ModelConverter#_get_field_class_for_property`). There's a way to solve that problem using the already existing `include_fk` and `exclude`. Here's a code snippet for you schema `Meta`, that uses the example for the documentation:  

``` python

class BookSchema(Schema):
    class Meta:
        model = Book
        include_fk = True
        exclude = tuple(prop.key
                        for prop in Book.__mapper__.iterate_properties
                        if hasattr(prop, 'direction'))

```

**Comment by diogommartins** [comment]
I recommend closing this issue or opening a PR to improve the documentation

**Comment by dusktreader** [comment]
Thanks for including the snippet above to accomplish fk inclusion without corresponding relationship properties. However, it's a bit clunky and it would certainly be nice to have that functionality wrapped up in its own property or method. I'd personally prefer that over just adding the snippet above to the documentation.

**Comment by diogommartins** [comment]
I agree with @dusktreader and i can submit a PR for that functionality. @sloria could you give us some directions ?

**Comment by cancan101** [comment]
I would love to see this as well and I agree that the snippet, while it works, is less than ideal.

**Comment by connebs** [comment]
I would also like to see this.

It defeats part of the purpose of `marshmallow-sqlalchemy` if I need to add/remove fields from `exclude` whenever I change the models.

Adding the above snippet to every one of my (many) schemas does not seem like a good solution.

**Comment by sloria** [comment]
Sorry for the delay in responding. I'm open to adding this. One workaround is to use `TableSchema` which both 

(1) skips over relationships and 
(2) deserializes to a dictionary instead of a model instance.

```python
class ArtistSchema(TableSchema):
    class Meta:
        table = models.Artist.___table__
```

This is a bit kludgy because you may want (1) without (2), but it's cleaner if you need both.

**Comment by sloria** [comment]
If anyone wants to take a stab at adding a new `include_relationships` option, I'd certainly review and merge it.

**Comment by sloria** [comment]
I'm working on this as part of https://github.com/marshmallow-code/marshmallow-sqlalchemy/pull/274

**Comment by sloria** [comment]
Released in 0.22.0

**Comment by zedrdave** [comment]
Sorry to reopen this, but… is there any reason why we can't have the schema generated with `dump_only` for relationship and `load_only` for foreign keys, by default? That would seem like the most likely use-cases.

Of course, this can be manually added, but seems to be defeating the whole purpose…
